authors/apps/comments/views.py

```diff
@@ -210,7 +202,8 @@ def patch(self, request, pk):
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response({"reply": serializer.data,
-                         "message": "Successfully edited reply: {}".format(pk)},
+                         "message": "Successfully edited reply: {}".format(pk)
```

Comment:
Hey @FahdJamy , I have seen above that you have added a comparison of the comment body on record with the new edited body. That's impressive. Did you just forget to do the same for the reply?

Reply:
Thanks for these observations again. Going to work on this.